stix_shifter_modules/infoblox/stix_translation/query_constructor.py

```diff
@@ -65,42 +65,14 @@ class QueryStringPatternTranslator:
     :param time_range: Default time range to query over (in minutes).
     :type time_range: int
 
-
     Attributes:
         comparator_lookup (map): Mapping, per dialect, of the ANTLR comparison operators and their native equivalent. Used in the tranlation process.
         qualified_queries (list): List of Infoblox translated & formatted queries. Queries are simply the HTTP query parameters for the specific APIs.
     """
-    comparator_lookup = {
-        'tideDbData': {
-            ComparisonExpressionOperators.And: "&",
-            ObservationOperators.And: '&',
-            ComparisonExpressionOperators.Or: "&",
-            ObservationOperators.Or: '&',
-            ComparisonComparators.Equal: "=",
-            ComparisonComparators.GreaterThan: "=",
```

Comment:
Infoblox seems to be the only reason you need to allow for dialect-specific operator files. But looking at this original mapping, this seems to be wrong and misleading. GreaterThan, LessThan, and so on are not the same as equals and would be misleading if we treat them as such in the translated query. Would it not have been better to just keep them as unsupported operators or am I missing something? I know you didn't develop this connector, this just jumped out at me is all. I think this connector should have just used the one set of operators and have it a smaller set at that. It also seems like the data source doesn't support any sort of OR operator.